app/assets/javascripts/components/enroll/enroll_card.jsx

```diff
@@ -74,6 +79,32 @@ const EnrollCard = ({
         course.passcode !== '' && <a href={courseLink} className="icon-close-small" />
       }
       {messageBody}
+      {modalShown
```

Comment:
I think this new behavior should be limited to the WikiEd configuration (ie, `Features.wikiEd` is truthy), with the current behavior remaining otherwise. The modal has a mix of Wikipedia rules and some advice that is given from the perspective of Wiki Education for student editors, so it might be a little bit of an overreach to build that advice in for Programs & Events Dashboard.